tools/automation/sections/building_section.py

```diff
@@ -0,0 +1,33 @@
+"""
+  A collection of classes to represent the building section of the YAML.
+
+  They are responsible for filling up the section with data.
+  
+  Holds logic for reading input files to retrieve the data for a building.
+"""
+
+from building_blocks.building import Building
+
+
+class DBOBuildingSection:
+
+  def __init__(self, files):

```

Comment:
nit: can we annotate what `files` are?

https://www.python.org/dev/peps/pep-3107/